source/test/stryker-config.json

```diff
@@ -8,7 +8,7 @@
 		"threshold-low": 60,
 		"threshold-break": 0,
 		"ignore-methods": [
-			"ToString",
```

Comment:
I guess we can discuss, if we want to include `ToString` again to be mutated.

Reply:
we decided to include mutations of `ToString` methods